app/views/users/registrations/edit.html.erb

```diff
@@ -1,57 +1,65 @@
 
-  <div class="field">
-    <%= f.label :current_password %>
-    <i>(we need your current password to confirm your changes)</i><br/>
-    <%= f.password_field :current_password, autocomplete: "current-password", class:"form-control" %>
-  </div>
+      <fieldset disabled>
+        <div class="form-group">
+          <%= f.label :email, "Email", class: "form-label" %>
+          <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "form-control", placeholder: "Enter email" %>
+        </div>
+      </fieldset>
 
-  <div class="actions">
-    <%= f.submit "Update", class:"btn btn-primary" %>
-  </div>
-<% end %>
+      <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
+        <div class="alert alert-warning">
+          Currently waiting confirmation for: <%= resource.unconfirmed_email %>
+        </div>
+      <% end %>
 
+      <div class="form-group">
+        <%= f.label :password, "Password", class: "form-label" %>
+        <i class="text-muted">(leave blank if you don't want to change it)</i>
+        <%= f.password_field :password, autocomplete: "new-password", class: "form-control", placeholder: "Enter new password" %>
+        <% if @minimum_password_length %>
+          <small class="form-text text-muted"><%= @minimum_password_length %> characters minimum</small>
+        <% end %>
+      </div>
 
+      <div class="form-group">
+        <%= f.label :password_confirmation, "Confirm Password", class: "form-label" %>
+        <%= f.password_field :password_confirmation, autocomplete: "new-password", class: "form-control", placeholder: "Confirm new password" %>
+      </div>
+
+      <div class="form-group">
+        <%= f.label :current_password, "Current Password", class: "form-label" %>
+        <i class="text-muted">(we need your current password to confirm your changes)</i>
+        <%= f.password_field :current_password, autocomplete: "current-password", class: "form-control", placeholder: "Enter current password" %>
+      </div>
+
+      <div class="form-group text-center">
+        <%= f.submit "Update", class: "btn btn-primary" %>
+      </div>
+    <% end %>
+
+ 
+
+    <%= link_to "Back", :back, class: "btn btn-secondary mt-3" %>
+  </div>
+</div>
 
-<div>Unhappy?
-  <h3>Cancel my account</h3>
-  <%= button_to "Cancel my account",
-  registration_path(resource_name),
-  data: {
-    confirm: "Are you sure?",
-    turbo_confirm: "Are you sure?",
-  },
-  method: :delete,
-  class: "btn btn-danger" %></div>
 
-<%= link_to "Back", :back %>
+   <div class="card mt-4 text-center border-danger">
+      <div class="card-body">
+        <h3 class="card-title text-danger my-4">Cancel My Account</h3>
+        <p class="card-text">
+          If you cancel your account, all your data will be permanently deleted.
+        </p>
+        <!-- Button trigger modal -->
+        <button type="button" class="btn btn-danger mb-2" data-toggle="modal" data-target="#exampleModalCenter">
```

Comment:
After removing the button_to in the prev code, where does the new modal button link to?